pig-runtime/src/org/partiql/pig/runtime/IonElementHelpers.kt

```diff
@@ -32,7 +33,7 @@ fun IonElement.toIonElement() = this
  * convert (or not) a variable to an [IonElement].  This has the same signature as the other `.toIonElement()`
  * extensions in package.
  */
-fun String?.toIonElement() = ionSymbol(this)
+fun String?.toIonElement() = this?.let { ionSymbol(this) } ?: ionNull()
```

Comment:
This looks more verbose now. Is it because `ionSymbol` had this code before?

Reply:
Yes.  the factory function `ionSymbol` used to return an ion null when passed a null value.  But I deemed that to be a little redundant.